tests/s3/test_object_tagging.py

```diff
@@ -1653,6 +1590,7 @@ def test_mpupload_tagkey128chars_42777(self):
         self.log.info("Verify Multipart Upload with tag having "
                       "tag key up to 128 Unicode characters in length")
 
+    @pytest.mark.parallel
```

Comment:
Let's not introduce this marker, unless you have verified parallel execution of the tests